benchto-driver/src/main/java/com/teradata/benchto/driver/loader/BenchmarkLoader.java

```diff
@@ -172,10 +177,33 @@ public void setup()
         }
 
         List<Path> benchmarkFiles = benchmarkFilesBuilder.build();
-        benchmarkFiles.stream().forEach((path) -> LOGGER.info("Benchmark found: {}", path.toString()));
+        benchmarkFiles.stream().forEach(path -> LOGGER.info("Benchmark found: {}", path.toString()));
         return benchmarkFiles;
     }
 
+    private void verifyNoNestedBenchmarkDirs()
+    {
+        boolean containsNestedBenchmarkDirs = properties.benchmarksFilesDirs().stream()
+                .filter(benchmarkFilesPath -> properties.benchmarksFilesDirs().stream()
+                        .filter(path -> !path.equals(benchmarkFilesPath) && path.startsWith(benchmarkFilesPath))
+                        .findAny().isPresent())
+                .findAny().isPresent();
```

Comment:
I do not like nested streams... do you find it more readable than simple `for` loop?